cli/src/alluxio.org/cli/cmd/info/collect.go

```diff
@@ -79,9 +78,6 @@ https://docs.alluxio.io/os/user/edge/en/operation/Troubleshooting.html#collect-a
 	cmd.Flags().StringSliceVar(&c.includeLogs, "include-logs", nil, "File name prefixes from ${ALLUXIO_HOME}/logs to include in the tarball, ignoring the default log files; cannot be used with --exclude-logs or --additional-logs")
 	cmd.Flags().BoolVar(&c.local, "local", false, "True to only collect information from the local machine")
 	cmd.Flags().IntVar(&c.maxThreads, "max-threads", 1, "Parallelism of the command; use a smaller value to limit network I/O when transferring tarballs")
-	const outputPath = "output-path"
```

Comment:
can we keep the `--output-dir` args here? i think it's possible to change the `cmdLineToArgs` to retrieve the output dir from args.